src/goto-instrument/cover_instrument_location.cpp

```diff
@@ -17,12 +17,12 @@ Author: Daniel Kroening
 void cover_location_instrumentert::instrument(
   goto_programt &goto_program,
   goto_programt::targett &i_it,
-  const cover_basic_blockst &basic_blocks) const
+  const cover_blocks_baset &basic_blocks) const
 {
   if(is_non_cover_assertion(i_it))
     i_it->make_skip();
 
-  const unsigned block_nr = basic_blocks.block_of(i_it);
+  const std::size_t block_nr = basic_blocks.block_of(i_it);
```

Comment:
Wrong commit?

Reply:
I added an initial commit for these changes